### PR TITLE
Add v0.22 schemas

### PR DESCRIPTION
**Pull request by Buckram123**

### CODE REVIEW

File: abstract/version-control/0.22.0/abstract-version-control.json
Comment (adairrr) [inline abstract/version-control/0.22.0/abstract-version-control.json:1]: Doesn't seem that we have all the schemas for Abstract-native modules?

**Review by adairrr — APPROVED** [review]

File: abstract/version-control/0.22.0/abstract-version-control.json
Comment (Buckram123) [reply abstract/version-control/0.22.0/abstract-version-control.json:1]: Which are missing?

Comment (Buckram123) [reply abstract/version-control/0.22.0/abstract-version-control.json:1]: Hm, ibc seems to be missing

Comment (Buckram123) [reply abstract/version-control/0.22.0/abstract-version-control.json:1]: > Hm, ibc seems to be missing

Nvm missed it

**Pull request merged**